ISHPullUp/ISHPullUpViewController.m

```diff
@@ -602,6 +611,10 @@ - (void)updateLayoutOfDimmingView:(UIView *)dimmingView bottomHeight:(CGFloat)bo
 }
 
 - (void)handleDimmingViewTapGesture:(UITapGestureRecognizer *)tap {
+    if (self.isLocked) {
```

Comment:
I thought this was nicer since the change is minimal, but we could also make it consistent with the other gesture.